doc/OnlineDocs/working_abstractmodels/data/index.rst

```diff
@@ -1,51 +1,50 @@
 .. _page-managingdata:
 
-Advanced Topics: Managing Data in AbstractModels
-================================================
+Managing Data in AbstractModels
+===============================
 
 There are roughly three ways of using data to construct a Pyomo
 model:
 
-1. use standard Python data objects, 
+1. use standard Python objects,
 
-2. initialize a model with data loaded with a :class:`DataPortal <ref pyomo.core.base.DataPortal.DataPortal>` object, and
+2. initialize a model with data loaded with a :class:`DataPortal <ref pyomo.environ.DataPortal>` object, and
```

Comment:
Same comment as before, did you mean to change the formatting of this class reference?